R/core.R

```diff
@@ -434,6 +434,25 @@ need_not_zero <- function(input, inputname) {
   }
 }
 
+#' @title update_min
+#'
+#' @rdname update_min
+#'
+#' @description Automatically updates numericInput to zero if input is negative.
+#'
+#' @param input numericInput field.
+#' @param session Current session.
+#'
+#' @return Zero value.
+#' @export
+update_min <- function(input, name, session) {
```

Comment:
Also consider calling it explicitly about negative or exposing the min value

```suggestion
update_min <- function(inputId, input, session) {
```
so that you can call it as 
```r 
update_min("salary", input, session)
```
instead of
``` r
update_min(input$salary, "input$salary", session)
```

Reply:
resolved with new commits